src/pages/signin/SignInPage.js

```diff
@@ -116,7 +116,7 @@ class SignInPage extends Component {
                     ) : (
                         <PasswordForm isVisible={showPasswordForm} />
                     )}
-                    {showResendValidationForm && <ResendValidationForm />}
+                    {Boolean(showResendValidationForm) && <ResendValidationForm />}
```

Comment:
Is this needed? `showResendValidationForm` is supposed to be a boolean.

Reply:
Maybe we should add Boolean() here, login is a String. https://github.com/Expensify/App/blob/cb5b1da4b9c25a088a290a51560c839932a473d0/src/pages/signin/SignInPage.js#L84-L86

Reply:
Yeah let's do that @alexxxwork